omdb/src/bin/omdb/nexus.rs

```diff
@@ -0,0 +1,246 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! omdb commands that query or update specific Nexus instances
+
+use anyhow::Context;
+use chrono::SecondsFormat;
+use clap::Args;
+use clap::Subcommand;
+use nexus_client::types::ActivationReason;
+use nexus_client::types::BackgroundTask;
+use nexus_client::types::CurrentStatus;
+use nexus_client::types::LastResult;
+use tabled::Tabled;
+
+/// Arguments to the "omdb nexus" subcommand
+#[derive(Debug, Args)]
+pub struct NexusArgs {
+    /// URL of the Nexus internal API
+    nexus_internal_url: String,
+
+    #[command(subcommand)]
+    command: NexusCommands,
+}
+
+/// Subcommands for the "omdb nexus" subcommand
+#[derive(Debug, Subcommand)]
+enum NexusCommands {
+    /// print information about background tasks
+    BackgroundTask(BackgroundTaskArgs),
+}
+
+#[derive(Debug, Args)]
+struct BackgroundTaskArgs {
+    #[command(subcommand)]
+    command: BackgroundTaskCommands,
+}
+
+#[derive(Debug, Subcommand)]
+enum BackgroundTaskCommands {
+    /// Show documentation about background tasks
+    Doc,
+    /// Print a summary of the status of all background tasks
+    List,
+    /// Print human-readable summary of the status of each background task
+    Details,
+}
```

Comment:
Another thing I like about this tool is how it describes some of the implementation in a useful way, which is educational to folks observing the system. These types of subcommands are great for that.